wiki/Difficulties/en.md

```diff
@@ -11,13 +11,14 @@ tags:
 
 *Not to be confused with [Beatmaps](/wiki/Beatmaps).*
 
-A beatmap's **difficulty** describes the level of skill needed to complete it. The difficulty name and star rating of a beatmap usually give enough information to tell how difficult it is.
+A beatmap's **difficulty** describes the level of skill needed to complete the beatmap. The difficulty name and star rating of a beatmap usually describe the level of skill needed to pass the beatmap.
+![An example of how a beatmap's star rating is displayed in the game.](img/difficulty-example.jpg)
```

Comment:
an example showing multiple difficulties is probably better, and with a set that has diff names relating to their difficulty (easy normal hard etc)